src/models/query.js

```diff
@@ -0,0 +1,58 @@
+import QueryAggregate from '../../src/models/queryAggregate';
+
+class Query {
+    /**
+     * [constructor]
+     * @param  {String} name
+     * @return {Query Instance}
+     */
+    constructor(name = '') {
+      this.aggregates = [];
+      this.filters = [];
+      this.name = name;
+    }
+
+    /**
+     * [addAggregate]
+     * @param {Array} aggregate
+     */
+    addAggregate(aggregate) {
+      this.aggregates = this.aggregates.concat(aggregate instanceof QueryAggregate ? aggregate : []);
```

Comment:
?????? is this a test ??? if isn't a instance of, you change it for a empty array ??? without saying to the user ?